dotCMS/src/main/java/com/dotmarketing/business/RelationshipAPIImpl.java

```diff
@@ -1,14 +1,200 @@
 package com.dotmarketing.business;
 
 
+import com.dotcms.business.CloseDBIfOpened;
+import com.dotcms.business.WrapInTransaction;
 import com.dotcms.contenttype.business.RelationshipFactoryImpl;
+import com.dotcms.contenttype.model.type.ContentTypeIf;
+import com.dotmarketing.beans.Tree;
+import com.dotmarketing.exception.DotDataException;
+import com.dotmarketing.exception.DotHibernateException;
+import com.dotmarketing.portlets.contentlet.model.Contentlet;
+import com.dotmarketing.portlets.structure.model.Relationship;
+
+import java.util.List;
 
 // THIS IS A FAKE API SO PEOPLE CAN FIND AND USE THE RELATIONSHIPFACTORY
 public class RelationshipAPIImpl extends RelationshipFactoryImpl implements RelationshipAPI {
 
+    @WrapInTransaction
+    @Override
+    public void deleteByContentType(ContentTypeIf type) throws DotDataException {
+        super.deleteByContentType(type);
+    }
+
+    @CloseDBIfOpened
+    @Override
+    public Relationship byInode(String inode) {
+        return super.byInode(inode);
+    }
+
+    @CloseDBIfOpened
+    @Override
+    public List<Relationship> byParent(ContentTypeIf parent) throws DotHibernateException {
+        return super.byParent(parent);
+    }
+
+    @CloseDBIfOpened
+    @Override
+    public List<Relationship> byChild(ContentTypeIf child) throws DotHibernateException {
+        return super.byChild(child);
+    }
+
+    @CloseDBIfOpened
+    @Override
+    public List<Relationship> dbAll() throws DotHibernateException {
+        return super.dbAll();
+    }
+
+    @CloseDBIfOpened
+    @Override
+    public List<Relationship> dbAll(String orderBy, String contentTypeInode) throws DotHibernateException {
+        return super.dbAll(orderBy, contentTypeInode);
+    }
+
+    @CloseDBIfOpened
+    @Override
+    public Relationship byTypeValue(String typeValue) {
+        return super.byTypeValue(typeValue);
+    }
+
+    @CloseDBIfOpened
+    @Override
+    public List<Relationship> byContentType(ContentTypeIf type) throws DotDataException {
+        return super.byContentType(type);
+    }
+
+    @CloseDBIfOpened
+    @Override
+    public List<Relationship> byContentType(ContentTypeIf st, boolean hasParent) {
+        return super.byContentType(st, hasParent);
+    }
+
+    @CloseDBIfOpened
+    @Override
+    public List<Relationship> byContentType(ContentTypeIf type, String orderBy) {
+        return super.byContentType(type, orderBy);
+    }
+
+    @CloseDBIfOpened
+    @Override
+    public List<Contentlet> dbRelatedContent(Relationship relationship, Contentlet contentlet) throws DotDataException {
+        return super.dbRelatedContent(relationship, contentlet);
+    }
+
+    @Override
+    public List<Contentlet> dbRelatedContent(Relationship relationship, Contentlet contentlet, boolean hasParent) throws DotDataException {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this method to simply inherit it. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1185)